File: comms/email/fetch-email/fetch-email.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "TWILIO_ACCOUNT_SID",
        authToken: "TWILIO_AUTH_TOKEN",
    });
    await client.emails.fetch("comms_email_01h9krwprkeee8fzqspvwy6nq8");
}
main();

File: comms/email/fetch-email/fetch-email.go
package example

import (
    context "context"

    client "github.com/twilio/twilio-comms-go/twilio/client"
    option "github.com/twilio/twilio-comms-go/twilio/option"
)

func do() {
    client := client.NewWithOptions(
        option.WithBasicAuth(
            "TWILIO_ACCOUNT_SID",
            "TWILIO_AUTH_TOKEN",
        ),
    )
    client.Emails.Fetch(
        context.TODO(),
        "comms_email_01h9krwprkeee8fzqspvwy6nq8",
    )
}
